libs/telemetry/events/bundle_init.go

```diff
@@ -0,0 +1,23 @@
+package events
+
+// Corresponds to the `DatabricksCliBundleInitEvent` proto message in `databricks_cli_log.proto`
+// as of 20 Dec 2024.
+type BundleInitEvent struct {
+	// UUID associated with the DAB itself. This is serialized into the DAB
+	// when a user runs `databricks bundle init` and all subsequent deployments of
+	// that DAB can then be associated with this init event.
+	Uuid string `json:"uuid,omitempty"`
+
+	// Name of the template initialized when the user ran `databricks bundle init`
+	// This is only populated when the template is a first party template like
+	// mlops-stacks or default-python.
+	TemplateName string `json:"template_name,omitempty"`
```

Comment:
Can it actually be empty?

Reply:
It'll be either "custom" or one of the names of the built-in templates.

Reply:
There's no server-side validation that prevents it from being empty though. It's type is captured as a generic string.